src/viz-probation-population/VizProbationPopulation.js

```diff
@@ -0,0 +1,172 @@
+import PropTypes from "prop-types";
+import React from "react";
+import styled from "styled-components";
+import judicialDistrictsTopology from "../assets/maps/us_nd_judicial_districts.json";
+import {
+  DEFAULT_TENANT,
+  OTHER_LABEL,
+  TENANTS,
+  TOTAL_KEY,
+  OTHER,
+} from "../constants";
+import PopulationViz from "../population-viz";
+import StateMap from "../state-map";
+import { recordIsTotal } from "../utils";
+
+const ASPECT_RATIO = TENANTS[DEFAULT_TENANT].aspectRatio;
+
+const ProbationMapWrapper = styled.div`
+  align-items: center;
+  display: flex;
+`;
+
+const ParticipantCount = styled.text`
+  dominant-baseline: middle;
+  fill: ${(props) =>
+    // eslint-disable-next-line no-nested-ternary
+    props.active
+      ? props.theme.maps.default.fill
+      : props.hover
+      ? props.theme.colors.highlight
+      : props.theme.colors.programParticipation};
+  font: ${(props) => props.theme.fonts.displayMedium};
+  font-size: 20px;
+  pointer-events: none;
+  text-anchor: middle;
+`;
+
+const OtherWrapper = styled.button`
+  background: none;
+  border: 0;
+  color: ${(props) =>
+    props.active ? props.theme.colors.highlight : props.theme.colors.heading};
+  cursor: pointer;
+  padding: 16px;
+  margin-left: -16px;
+  text-align: center;
+
+  &:hover {
+    color: ${(props) => props.theme.colors.highlight};
+  }
+`;
+
+const OtherLabel = styled.div`
+  font: ${(props) => props.theme.fonts.body};
+`;
+
+const findDistrictRecord = (district) => (record) =>
+  record.district === district;
+
+function ProbationMap({ currentLocation, data, onLocationClick, width }) {
+  // eslint-disable-next-line react/prop-types
+  const DistrictLabel = ({ hover, locationId, objectId }) => {
+    const districtRecord = data.find(findDistrictRecord(objectId));
+    return districtRecord ? (
+      <ParticipantCount
+        active={locationId === districtRecord.district}
+        hover={hover}
+      >
+        {districtRecord.value}
+      </ParticipantCount>
+    ) : null;
+  };
+
+  const otherRecord = data.find(findDistrictRecord(OTHER));
+
+  return (
+    <ProbationMapWrapper>
+      <StateMap
+        aspectRatio={ASPECT_RATIO}
+        LabelComponent={DistrictLabel}
+        locationId={currentLocation}
+        onRegionClick={onLocationClick}
+        stateTopology={judicialDistrictsTopology}
+        width={width}
+      />
+      {otherRecord && (
+        <OtherWrapper
+          active={currentLocation === OTHER}
+          onClick={(e) => {
+            e.preventDefault();
+            onLocationClick(OTHER);
+          }}
+        >
+          <ParticipantCount as="div">{otherRecord.value}</ParticipantCount>
```

Comment:
Whoa, didn't know you could switch the element type on the fly like this.  I'm assuming any svg styles that get applied to `ParticipantCount` are just ignored here now that it's a `div`?

Reply:
yeah styled-components supports that ... CSS is pretty fault-tolerant so your browser should just ignore any properties that don't apply (because they are specific to SVG or whatever)